src/js/server/controllers/apply/concerns/appliable-concern.ts

```diff
@@ -0,0 +1,32 @@
+import { HackerApplicationInstance } from 'js/server/models';
+import * as statuses from 'js/shared/status-constants';
+
```

Comment:
I believe these functions are only used once so I'm not sure of the benefit of moving them into a concern.  (I presume you're using Rails-style nomenclature?)

Reply:
It was really just to keep them out of the router (which then just has the routes). I'm not attached to naming, I just thought middlewares seemed a bit generic, but since they're not imported by the controllers anyway it might make more sense to call them that and move them to a `/middlewares` folder.